src/ngraph/ngraph_emitter.cc

```diff
@@ -1038,11 +1038,22 @@ void Emitter::CreateLayerOps() {
     const NgraphNodePtr one = makeConstant(dtype, shape, 1.0f);
 
     NgraphNodePtr scale_grad;
+
+    #if MXNET_USE_NGRAPH_DISTRIBUTED
+    NgraphNodePtr ng_grad = std::make_shared<ngraph::op::AllReduce>(grad);
```

Comment:
the name `ng_grad` confused me a little bit. Does the `ng` stands for `ngraph`? 
I wonder if we could do reassign like this:
`grad = std::make_shared<ngraph::op::AllReduce>(grad);`? Then the snippet below could be reused too.